app/plugin.go

```diff
@@ -301,6 +326,10 @@ func (a *App) EnablePlugin(id string) *model.AppError {
 		return model.NewAppError("EnablePlugin", "app.plugin.not_installed.app_error", nil, "", http.StatusBadRequest)
 	}
 
+	if err := a.activatePlugin(manifest); err != nil {
+		return err
+	}
```

Comment:
Added this so that the API would actually report if the plugin failed to activate. Previously we'd just update the config and the activation would happen implicitly